2-mandatory/2-financial-times.js

```diff
@@ -5,16 +5,43 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
-}
+    const newarray = [];
+
+      for (const item of allArticleTitles) {
+        if (item.length <= 65) {
+          newarray.push(item);
+        }
+      }
+      return newarray;
+    }
+
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+
+    let fewestWord=ARTICLE_TITLES[0].split(' ').length;
+    let finalTitle=ARTICLE_TITLES[0];
+    let i=1;
+
+    while(i<ARTICLE_TITLES.length){
```

Comment:
In this while loop, you are using 'i' as an iterator to go on loop until it reaches the condition  i < ARTICLE_TITLES.length. 

Is there another way to improve this loop so it's more clear what it's doing?

Reply:
while(i=1, i<ARTICLE_TITLES.length, i++)